android-plugin/src/main/java/io/ecocode/java/Java.java

```diff
@@ -20,11 +20,11 @@
 package io.ecocode.java;
 
 public interface Java { //NOSONAR - we use the interface for constant without inheriting it
-    String REPOSITORY_NAME = "ecoCode";
-    String PROFILE_NAME = "ecoCode";
+    String REPOSITORY_NAME = "ecoCode (Android)";
```

Comment:
why don't we add "Java" key word ?

Reply:
I don't think it's necessary because the language is already added as a suffix:
<img width="283" alt="image" src="https://github.com/green-code-initiative/ecoCode-android/assets/51145/5b5f029e-8786-44af-bc4a-605ad2f119f1">